### ISSUE TITLE
Scrambled Midnight Commander output

### ISSUE THREAD

**Issue by countfuzzball** [Swordfish90/cool-retro-term#519]
I'm getting scrambled Midnight Commander output from CRT, as in the screenshot below.

![mc1](https://user-images.githubusercontent.com/49041545/55186013-782d8b00-518d-11e9-8bd7-9fb995fb3363.jpg)

If I use the switch -a to Midnight Commander (disable usage of graphic characters for line drawing),
I get a somewhat usable interface like so: 

![mc2](https://user-images.githubusercontent.com/49041545/55186269-0a359380-518e-11e9-8f1f-c5e8aaa75df0.jpg)

Is my system misconfigured somehow?

**Comment by Swordfish90** [comment]
Does this happen with any font you choose?

**Comment by countfuzzball** [comment]
Yes. I've tried all the HD fonts, the "basic" ones and the System fonts too.

**Comment by countfuzzball** [comment]
Ok, so I asked around on my distro's forum and I got back the reply that the fonts I'm using are just missing the "CP437" symbols from the fonts themselves.
Which is weird, as I see on the readme that Midnight Commander works fine in CRT... so I'm not sure what I'm doing wrong on my end?

**Comment by sacredbanana** [comment]
I'd like this issue resolved too
